### ISSUE TITLE
Bad configuration files prevents opening new files

### ISSUE THREAD

**Issue by robmcmullen** [robmcmullen/peppy#759]
_[Trac time 20090822 040614Z]_ A bad recentfiles.txt causes traceback:

```
Traceback (most recent call last):
File wx\_core.pyo, line 14618, in
File peppy\fileopener.pyo, line 244, in finalizeAfterSuccessfulLoad File peppy\
otebook.pyo, line 285, in newMode 
File peppy\
otebook.pyo, line 200, in updateWrapper 
File peppy\frame.pyo, line 693, in switchMode 
File peppy\frame.pyo, line 471, in setMenumap 
File peppy\menu.pyo, line 305, in updateActions 
File peppy\menu.pyo, line 391, in updateMenuAccelerators 
File peppy\actions\__init__.pyo, line 890, in insertIntoMenu 
File peppy\actions\__init__.pyo, line 695, in insertIntoMenu 
File peppy\actions\__init__.pyo, line 698, in _insertItems 
File peppy\mainmenu.pyo, line 850, in getItemsAttributeError: NoneType object has no attribute getKeywordOrder
```

**Comment by robmcmullen** [comment]
_[Trac time 20090822 043430Z]_ a1324066f2b1b30ba5f3e8df8452b4651a3f66f8 Refs #759: found problem when recentfiles.txt contains blank line
- would crash with invalid menu item, so now check to make sure line is non-blank

**Comment by robmcmullen** [comment]
_[Trac time 20090822 144218Z]_ Not just limited to recentfiles.txt; also found that a blank editra style sheet in preferences.cfg can cause the same type of problem.

**Comment by robmcmullen** [comment]
_[Trac time 20090822 151738Z]_ 3c367767e8b472ca7bdd6f08e0dc14a092859b84 Fixed #759: added try/catch around editra style sheet failure
- moved editra styling into applyEditraStyling and on exception resets the styling to the built-in default style sheet
